Dungeoneer/src/com/interrupt/dungeoneer/entities/Prefab.java

```diff
@@ -49,7 +49,7 @@ public void SpawnEntity(Level level, Entity e) {
 		if (e instanceof Monster) {
 			((Monster)e).Init(level, Game.instance.player.level);
 		}
-		level.addEntity(e);
```

Comment:
Changing the Prefab behavior is potentially scary, could this be wired up by overriding the `tossItem` function like the Potion does?

https://github.com/Interrupt/delverengine/blob/18a41f22f4e72e2970e84ea14e6cae6722b42de4/Dungeoneer/src/com/interrupt/dungeoneer/entities/items/Potion.java#L237

Reply:
This is for when you break a pot and bomb drops. Instead of changing how prefabs spawn things, I could change the guard inside of `FusedBomb`